snippets/c/c.json

```diff
@@ -398,10 +388,10 @@
         "description": "Error checking for pointer-type IO functions"
     },
     "fseek() errcheck": {
-        "prefix": "chkio_m1",
+        "prefix": "chkio_mi",
```

Comment:
What's the reasoning behind this prefix change?

Reply:
I removed numbers (and punctuation) in the prefixes, because they can unintentionally make the completion menu pop up; nvim-cmp does that when I type just a number or a comma. This is very annoying because I would need to close the menu first before e.g. typing **enter** or otherwise a snippet will get inserted and I would have to undo everything :anger: :rage:...

So, the `m1` had to be replaced with something else representing **minus-one**. Something like `mone` would be too long and look weird (most ppl would read it as **mo-ne** instead of **m-one** and get confused), so I took the `i` from `ifi`, because it looks similar to `1`, plus didn't make the snippet longer.

Reply:
> have to undo everything 💢 😡...
I feel this about some of my own local snippets :joy: 

Thanks for the explanation, LGTM !